tests/network/network.go

```diff
@@ -537,3 +550,12 @@ func (n *LocalNetwork) GetTwoL1s() (
 	Expect(len(l1s)).Should(BeNumerically(">=", 2))
 	return l1s[0], l1s[1]
 }
+
+// TODO: once tmpnet supports static port option across restarts, remove this function
```

Comment:
Is this something that is planned to be added on tmpnet? I forget the context on that side, but I thought this support was actually removed since it's no longer needed.

Reply:
Yeah -- it's already added back but requiring explicit configuration:  https://github.com/ava-labs/avalanchego/pull/3697 not including it in this PR since would need to be on branches again and this PR allows us to use properly tagged releases.